### DISCUSSION TITLE
Is it possible to map the www-data user in the container to a different UID on the host?

### DISCUSSION THREAD

**Discussion by relink2013** [nextcloud/all-in-one#5799] [Questions] [answered]
I’m going to give Nextcloud another go and I plan on making heavy use of “external storage”. 

Is there any way to essentially have docker translate the `uid` so the data on the host is owned by `uid99` but inside the container Nextcloud sees it as `uid33`. 

This way Nextcloud and the host OS are both happy and I can leave my permissions alone.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1974
